examples/08_experimental/plot_two_runs_model_experimental.py

```diff
@@ -0,0 +1,165 @@
+"""
+Simple example of two-runs fMRI surface based model fitting
```

Comment:
Maybe more interesting as a new example rather than redoing what exist in volume space would be to compare volume and surface based GLM.

Also good for the devs to have some visuals to make sure both approaches do not give wildely different results.